srcpkgs/ibus-chewing/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'ibus-chewing'
+pkgname=ibus-chewing
+version=1.5.1
+revision=1
+archs="x86_64"
+build_style=gnu-makefile
+hostmakedepends="git cmake pkg-config"
+makedepends="gettext-devel libchewing-devel ibus-devel gob2 libX11-devel gtk+3-devel glib-devel findutils"
```

Comment:
`makedepends` is for library dependencies, `hostmakedepends` is for application dependencies, `gettext-devel` should be in `hostmakedepends`, I imagine `findutils` should be as well.